Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -280,4 +280,12 @@ lemma mem_specialOrthogonalGroup_fin_two_iff {M : Matrix (Fin 2) (Fin 2) R} :
 
 end specialOrthogonalGroup
 
+open scoped Kronecker in
+theorem UnitaryGroup.kronecker_mem_unitaryGroup {R m : Type*} [Fintype m] [DecidableEq m]
+    [CommRing R] [StarRing R] (U₁ : unitaryGroup n R) (U₂ : unitaryGroup m R) :
```

Comment:
I think this should be stated as `U₂ ∈ unitaryGroup m R` not `U₂ : unitaryGroup m R`, as the result would then be syntactically more general.